core/modules/parser/QSMySqlListener.cc

```diff
@@ -2056,30 +2056,33 @@ class SimpleIdAdapter :
 
 
 class DottedIdAdapter :
-        public AdapterT<DottedIdCBH, QSMySqlParser::DottedIdContext> {
+        public AdapterT<DottedIdCBH, QSMySqlParser::DottedIdContext>,
```

Comment:
A brief explanation of what this class does or its purpose could be useful.

Reply:
There is an explanation of Adapter in the base class (this is an AdapterT, which is an Adapter). There are many specializations of Adapter (one per grammar element) and it would be repetitive to explain each one.